### ISSUE TITLE
GridFS upload freezes if chunkSizeBytes is not specified in 2.0

### ISSUE THREAD

**Issue by Frederick888** [doctrine/mongodb-odm#2087]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | no
| Version     | 2.0.1

#### Summary
GridFS upload freezes if `chunkSizeBytes` is not explicitly specified in File model annotation.

#### Current behavior
It freezes in `\MongoDB\GridFS\Bucket::uploadFromStream` at `stream_copy_to_stream` (FYI I'm using `mongodb/mongodb` 1.5.0).

The reason why this happens is that `\Doctrine\ODM\MongoDB\Mapping\ClassMetadata::getChunkSizeBytes` returns `null` when `chunkSizeBytes` is not specified, and it's not overridden by the default value at `\MongoDB\GridFS\Bucket::openUploadStream` (`$options += ['chunkSizeBytes' => $this->chunkSizeBytes]`).

A possible solution is to avoid the `null` value in `\Doctrine\ODM\MongoDB\Repository\DefaultGridFSRepository::prepareOptions`.

#### How to reproduce
Here's a simple unit test I put together.
```php
<?php

declare(strict_types=1);

namespace Doctrine\ODM\MongoDB\Tests\Functional\Ticket;

use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
use Doctrine\ODM\MongoDB\Repository\GridFSRepository;
use Doctrine\ODM\MongoDB\Tests\BaseTest;

class GH2087Test extends BaseTest
{
    public function testFallbackGridFSChunkSize()
    {
        /** @var GridFSRepository $repository */
        $repository = $this->dm->getRepository(GH2087File::class);
        /** @var GH2087File $file */
        $file = $repository->uploadFromFile(__FILE__);
        $this->assertNotNull($file->_id);
    }
}

/**
 * @ODM\File(bucketName="test")
 *
 * Class GH2087File
 * @package Doctrine\ODM\MongoDB\Tests\Functional\Ticket
 */
class GH2087File
{
    /**
     * @ODM\Id
     * @var string
     */
    public $_id;
    /**
     * @ODM\File\Filename
     * @var string
     */
    public $filename;
    /**
     * @ODM\File\Length
     * @var int
     */
    public $length;
    /**
     * @ODM\File\UploadDate
     * @var \DateTimeInterface
     */
    public $uploadDate;
    /**
     * @ODM\File\ChunkSize
     */
    public $chunkSize;
}
```

#### Expected behavior
File uploads normally without explicitly specified chunk size in model (falls back to default).

**Comment by blakyris** [comment]
Same for the `uploadFromFile();` method if I do like this :

``$file = $this->documentManager->getRepository(File::class)->uploadFromFile($path);

It freezes. Nothing happens in the Symfony CLI

**Comment by alcaeus** [comment]
Fixed in #2094.
